client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -414,8 +414,8 @@ const PatternAssembler = ( {
 							globalStyles: syncedGlobalStylesUserConfig,
 							// Newly created sites should reset the starter content created from the default Headstart annotation
 							shouldResetContent: isNewSite,
-							// Also, new sites except for virtual themes set the option wpcom_site_setup=assembler
-							siteSetupOption: isNewSite && ! design.is_virtual ? 'assembler' : '',
```

Comment:
Any background discussion / decision on why this `isNewSite` clause is removed from the predicate? 😄

Reply:
pekYwv-1YV-p2#comment-1557